packages/react/src/create-strict-api/__tests__/index.test.tsx

```diff
@@ -79,7 +86,7 @@ describe('createStrictAPI()', () => {
         color: 'var(--ds-text)',
         all: 'inherit',
         '&:hover': { color: 'var(--ds-text-hover)' },
-        '&:invalid': { color: 'orange' },
```

Comment:
Inside the `&:invalid { }` block, if I had a property or value that wasn't one of these

```
  color: 'var(--ds-text)';
  background: 'var(--ds-surface)' | 'var(--ds-surface-sunken)';
  bkgrnd: 'red' | 'green';
```
(according to the source code in https://github.com/atlassian-labs/compiled/blob/fe384d5fc2d1f52cd2dba93937b5c85e013fdf53/packages/react/src/create-strict-api/__tests__/__fixtures__/strict-api.ts#L3 ) , would we have a type error?

I noticed that `&:invalid` isn't explicitly specified in `packages/react/src/create-strict-api/__tests__/__fixtures__/strict-api.ts`, so not sure how the typechecking should work inside `&:invalid { ... }`

Reply:
This is the new behaviour — values now resolve to the top level ones unless there is a more specific one defined (inside an object in the schema e.g. `&:invalid: {}`).

Reply:
ahh awesome 👍

Reply:
@kylorhall-atlassian I've marked this as a breaking change. Gut feel was it's breaking a bunch of behaviour. Or at least you'll get some type errors. I could go both ways, wdyt?

Reply:
I don't think it hurts anyone to call it a breaking change, as for us, if this was used, it's a change that would be noisy or difficult to adopt.  Again, I think it's a breaking change that will impact no one.